<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCalculator</class>
 <widget class="QWidget" name="CCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QTextEdit" name="textEditMain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>111</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>202</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_lnv">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>lnv</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_pi">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>π</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_e">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>e</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_C">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_back">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_pow2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_complement">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_abs">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>|x|</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_exp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>exp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_mod">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>mod</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_sqrt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>2√x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_lpart">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_rpart">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_factorial">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>n!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_div">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_mi">
        <property name="text">
         <string>x^y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_mul">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_10mi">
        <property name="text">
         <string>10^x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_sub">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_log">
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_add">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Button_ln">
        <property name="text">
         <string>ln</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_re">
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_point">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button_equal">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>科学</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_trangle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>三角学 v</string>
   </property>
  </widget>
  <widget class="QFrame" name="Frame_Trangle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Button_hyp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>hyp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_sin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_cos">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_tan">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_sec">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_csc">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>csc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_cot">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>cot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_TrLnv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>lnv</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Button_function">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>函数 v</string>
   </property>
  </widget>
  <widget class="QFrame" name="Frame_Function">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Button_abs2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>|x|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_floor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>⌊x⌋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_ceil">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>⌈x⌉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_deg">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>→deg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_rand">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>rand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_dms">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>→dms</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
